File: 프로그래머스/0/181853. 뒤에서 5등까지/뒤에서 5등까지.cs
using System;
using System.Collections.Generic;
    
public class Solution 
{
    public int[] solution(int[] num_list) 
    {
        List<int> numList = new List<int>(num_list);
        numList.Sort();
        
        while(numList.Count != 5) {
            numList.RemoveAt(numList.Count-1);
        }
        return numList.ToArray();
    }
}

File: 프로그래머스/0/181853. 뒤에서 5등까지/뒤에서 5등까지.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> num_list) 
{
    vector<int> answer(5,0);
    sort(num_list.begin(),num_list.end());
    
    copy(num_list.begin(),num_list.begin()+6,answer.begin());
    return answer;
}